### ISSUE TITLE
[Win] Closing popout window (created through window.open) crashes application.

### ISSUE THREAD

**Issue by stewhouston** [jiahaog/nativefier#264]
### Description

When closing any popout window generated through the application, I receive an error prompt reading: 'A JavaScript error occurred in the main process', which subsequently crashes the entire application (requires the process to be terminated.)

Screenshot here: http://i.imgur.com/GlzFx3e.png
### Steps to Reproduce Issue

Open a popup created using window.open, then close it. Site being used: www.overcharge.tv (reddit links in the left hand column.)
### Specifications
- nativerfier 7.0.1
- node v6.2.0
- OS: Windows 10
- Architecture: AMD x64
- Error message with stack trace: http://i.imgur.com/GlzFx3e.png

```
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined at TypeError (native) at Object.defineProperty.set
```

**Comment by ronjouch** [comment]
@stewhouston thankd for the report! Before we investigate more, can you please try with the last nativefier and a recent electron? `nativefier --electron-version 1.6.6 "https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_win_open"` worked just fine for me.

**Comment by ronjouch** [comment]
Closing after two weeks as `need info` and successful tests under Windows 10 and the recently released nativefier 7.2.0 which defaults to Electron 1.6.6. @stewhouston feel free to chime in and re-open the discussion anytime!
